models/comment.py

```diff
@@ -0,0 +1,18 @@
+from sqlalchemy import Column, Integer, String, ForeignKey
+from models.database import Base
+
+
+class Comment(Base):
+    """
+    Comment
+    """
+    __tablename__ = "comments"
```

Comment:
SQL table names should be singular
```suggestion
    __tablename__ = "comment"
```